laterpay/__init__.py

```diff
@@ -472,3 +482,78 @@ def get_access(self, article_ids, product_key=None):
             raise Exception(data['status'])
 
         return data
+
+    def get_request_headers(self):
+        """
+        Return a ``dict`` of request headers to be sent to the API.
+        """
+        return {
+            'X-LP-APIVersion': 2,
+            # TODO: Automatic versioning.
+            'User-Agent': 'LaterPay Client - Python - v4.2.0'
```

Comment:
We should either mention this in "Release Checklist" in README or maybe try to use something like 
```
pkg_resources.get_distribution('laterpay-client').version
```